app/Repositories/CacheUserRepository.php

```diff
@@ -197,11 +233,23 @@ protected function retrieve($key)
      * Items not found in the cache will have a null value.
      *
      * @param  array  $keys
-     * @return array
+     * @return array|null
      */
     protected function retrieveMany(array $keys)
     {
-        return Cache::many($keys);
+        $retrieved = [];
+
+        $data = Cache::many($keys);
+
+        foreach ($data as $item) {
+            if ($item) {
+                $retrieved[] = $item;
+            }
+        }
+
+        if (count($retrieved)) {
+            return $data;
```

Comment:
@joe It actually does return `null` when the function doesn't return anything. But by being more explicit it would have avoided this whole convo which is why I was asking about it in our slack convo :wink: